schemas/event.yml

```diff
@@ -156,6 +156,19 @@
       normalize:
         - array
       allowed_values:
+        - name: api
+          description: >
+            Events in this category correspond to API events propagated directly
+            from the Operating System (Windows, Linux, etc.), from either the native
+            API function or system call, or a managed source of events (such as ETW, syslog).
+          expected_event_types:
```

Comment:
Narrowing down this list of expected `event.type` values is tough. I could see adding others, but it's also far easier to add others later than remove.

Maybe `info` for APIs that are purely returning information and not taking action or making a state change?

And perhaps also `creation`/`change`/`deletion` for the creating and removing of a resource by way of an API call?